apps/angular/pipe-intermediate/src/app/app.component.ts

```diff
@@ -1,34 +1,26 @@
 import { NgFor } from '@angular/common';
 import { Component } from '@angular/core';
+import { VerifyPipe } from './pipes/verify.pipe';
+
+export type Person = {
+  name: string;
+  age: number;
+};
 
 @Component({
   standalone: true,
-  imports: [NgFor],
+  imports: [NgFor, VerifyPipe],
   selector: 'app-root',
   template: `
     <div *ngFor="let person of persons; let index = index; let isFirst = first">
-      {{ showName(person.name, index) }}
-      {{ isAllowed(person.age, isFirst) }}
+      {{ person | verify : index : isFirst }}
```

Comment:
You didn't understand the challenge. You created a specific pipe for this specific use case. The goal of this challenge is to create a **generic** pipe that will work for any function and which is strongly typed

Reply:
got it @tomalaforge, will try to understand it properly.